DevCGlobalDirectory.md

```diff
@@ -17,6 +17,36 @@
   - Github: https://github.com/Facebook-Developers-Circle-CDMX-Retos
 </details>
 
+<details>
+  <summary>
+    <b>DevC Cordoba City</b>
+  </summary>
+
+  - Lead(s):    
+    - Juan Cruz Gabriel
```

Comment:
Thanks for this amazing PR!

Can you add the GitHub profile links of the leads, please?